--- a/proguard-rules.pro
+++ b/proguard-rules.pro
@@ -35,7 +35,10 @@
 -keepclassmembers class de.blinkt.openvpn.core.VPNLaunchHelper {public *;}
 -keep class de.blinkt.openvpn.core.VpnStatus$ByteCountListener
 -keep class de.blinkt.openvpn.core.VpnStatus$StateListener
+-keep class de.blinkt.openvpn.core.VpnStatus$LogListener
 -keep class de.blinkt.openvpn.core.VpnStatus
 -keepclassmembers class de.blinkt.openvpn.core.VpnStatus {public *;}
+-keep class de.blinkt.openvpn.core.LogItem
+-keepclassmembers class de.blinkt.openvpn.core.LogItem {public *;}
 -keep class de.blinkt.openvpn.utils.**
 -keepclassmembers class de.blinkt.openvpn.utils.** { public *;}
 -keep class de.blinkt.openvpn.core.keepVPNAlive
